### DISCUSSION TITLE
How to import mp3 files in NEXT.js?

### DISCUSSION THREAD

**Discussion by ShourovRoy** [vercel/next.js#25289] [Help]
in next.js I can't import the mp3 file or even it's not working from public dir.
I am making a notification when a message arrives. But facing this problem. I am trying to use the mp3 file using useSound hook.
here is the npm package: https://www.npmjs.com/package/use-sound
it's working with create-react-app. please help me to use it in nextjs.
![Screenshot_1](https://user-images.githubusercontent.com/68475757/118956760-b693ee80-b981-11eb-85f8-cfb48fc81e3a.png)

**Comment by wwwdepot** [comment]
If you put mp3 in the public directory, just use its public path. `public/your.mp3` becomes `/yours.mp3` in browser.

```js
const [play] = useSound("/your.mp3");
```

**Reply by ShourovRoy** [reply]
I tried it also not working
